vita3k/module/src/load_module.cpp

```diff
@@ -41,17 +41,19 @@ bool is_lle_module(SceSysmoduleModuleId module_id, EmuEnvState &emuenv) {
         SCE_SYSMODULE_HTTP,
         SCE_SYSMODULE_SSL,
         SCE_SYSMODULE_HTTPS,
+        SCE_SYSMODULE_SMART,
+        SCE_SYSMODULE_FACE
```

Comment:
so revert this

Reply:
This list i have created is only for fw module, not for sce_module loaded in pre load